lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -36,6 +36,7 @@ use constant CACHE_SERVICE_TEST_SYNC_ATTEMPTS => $ENV{OPENQA_CACHE_SERVICE_TEST_
 
 my $isotovideo = '/usr/bin/isotovideo';
 my $workerpid;
+my $isotovideo_interface_version = 0;
```

Comment:
It would be great to avoid introducing further global variables. This version number is already accessible via an attribute of the worker object (which is even already present in a local variable in the function where you need it).